winui/Shimmer/Customization.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Customization in WinUI Shimmer control | Syncfusion
+description: Learn how to customize the appearance of the Syncfusion WinUI Shimmer(SfShimmer) control using various properties like Fill, Wave Color, Wave Width, Repeat Count and Wave Duration.
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# Customization in WinUI Shimmer
+
+This section explains how to customize the appearance of the Syncfusion WinUI Shimmer (SfShimmer) control using various properties. You can modify the following aspects of the Shimmer effect:
```

Comment:
@GilbertmanojSF3971 This section explains how to customize the appearance of the Syncfusion WinUI Shimmer (SfShimmer) control using various properties. You will learn how to modify the following aspects of the shimmer effect: